babel.config.js

```diff
@@ -1,7 +1,6 @@
-module.exports = {
-  retainLines: true,
-  compact: true,
-  comments: false,
-  presets: ["module:metro-react-native-babel-preset"],
-  sourceMaps: false
+module.exports = function(api) {
```

Comment:
I'm not fully familiar with these changes -- is it a guarantee that `module.exports = function` is always going to be called by babel? Is there any documentation you can link me to to understand what this is doing?

Reply:
yes, this is common practice. Please refer to the official documentation https://babeljs.io/docs/en/configuration